windows/searchlibrary/0f7750bd-1576-481a-b60d-b382bc9e70fd.query

```diff
@@ -6,10 +6,10 @@
 tag=$WINDOWS_LOGON winlog TimeCreated EventID == 4624 Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType LogonProcessName WorkstationName as TargetInfo IpAddress
 | lookup -s -r @groups [Computer SubjectUserName SubjectDomainName TargetUserName TargetDomainName LogonType] [Comp SUN SDN TUN TDN LT]
 | lookup -r windows_login_types LogonType logon_type name as LogonName
-| dns -reverse IpAddress Host
-| printf -e SourceInfo "%v (%v)" Host IpAddress
+/* | dns -reverse IpAddress Host */
+| printf -e SourceInfo "%v" IpAddress

```

Comment:
just use `IpAddress` directly in the query, don't format it as a new name, this is just load and achieves nothing.